tests/trainers/test_finetune_vision_efficient_tuning_swift.py

```diff
@@ -4,12 +4,10 @@
 import tempfile
 import unittest
 
+from swift import AdapterConfig, LoRAConfig, PromptConfig
```

Comment:
move test case to swift

Reply:
moved

Reply:
The case uses efficient trainer, so basicly it's a test case belonged here
So instead, i made the test case optional to run.